addons/point_of_sale/tests/test_report_session.py

```diff
@@ -112,3 +112,88 @@ def test_report_session_2(self):
 
         pdf = self.env['ir.actions.report']._render_qweb_pdf('point_of_sale.sale_details_report', res_ids=session_id_2)
         self.assertTrue(pdf)
+
+    def test_report_listing(self):
+        product1 = self.create_product('Product 1', self.categ_basic, 150)
+        product2 = self.create_product('Product 2', self.categ_basic, 150)
+
+        cash_payment_method = self.env['pos.payment.method'].create({
+            'name': 'Cash',
+            'receivable_account_id': self.company_data['default_account_receivable'].id,
+            'journal_id': self.company_data['default_journal_cash'].id,
+            'company_id': self.env.company.id,
+        })
+        bank_payment_method = self.env['pos.payment.method'].create({
+            'name': 'Bank',
+            'journal_id': self.company_data['default_journal_bank'].id,
+            'receivable_account_id': self.company_data['default_account_receivable'].id,
+            'company_id': self.env.company.id,
+        })
+        self.config.write({'payment_method_ids': [(4, bank_payment_method.id), (4, cash_payment_method.id)]})
+
+        self.open_new_session()
+        session = self.pos_session
+
+        order = self.env['pos.order'].create({
+            'session_id': session.id,
+            'lines': [(0, 0, {
+                'name': "TR/0001",
+                'product_id': product1.id,
+                'price_unit': 150,
+                'discount': 0,
+                'qty': 1.0,
+                'price_subtotal': 150,
+                'price_subtotal_incl': 165,
```

Comment:
Nitpicking to stay coherent with tax information:
```suggestion
                'qty': 1.0,
                'price_subtotal': 150,
                'tax_ids': [(6, 0, self.tax_sale_a.ids)],
                'price_subtotal_incl': 165,
```
and you can add `self.tax_sale_a['amount'] = 10` before creating the order.